From 0352247123bccaf0d8f0eaacfdd0c17c6331e6fc Mon Sep 17 00:00:00 2001
From: Alessandro Decina <alessandro.decina@collabora.co.uk>
Date: Thu, 22 Sep 2011 18:53:15 +0200
Subject: [PATCH 088/154] pvrvideosink: restrict input to yuv-strided for now

YUV and RGB work but they are not very stable yet
---
 sys/pvr2d/gstpvrvideosink.c |   16 +++++++---------
 1 files changed, 7 insertions(+), 9 deletions(-)

diff --git a/sys/pvr2d/gstpvrvideosink.c b/sys/pvr2d/gstpvrvideosink.c
index 4c39638..23eb4cf 100644
--- a/sys/pvr2d/gstpvrvideosink.c
+++ b/sys/pvr2d/gstpvrvideosink.c
@@ -67,17 +67,15 @@ static void gst_pvrvideosink_xwindow_destroy (GstPVRVideoSink * pvrvideosink,
     GstXWindow * xwindow);
 
 static GstStaticPadTemplate gst_pvrvideosink_sink_template_factory =
-    GST_STATIC_PAD_TEMPLATE ("sink",
+GST_STATIC_PAD_TEMPLATE ("sink",
     GST_PAD_SINK,
     GST_PAD_ALWAYS,
-    GST_STATIC_CAPS (
-        "video/x-raw-yuv-strided, "                \
-        "format = (fourcc) NV12, "                 \
-        "rowstride = (int) 4096, "                 \
-        "width = " GST_VIDEO_SIZE_RANGE ", "       \
-        "height = " GST_VIDEO_SIZE_RANGE ", "      \
-        "framerate = " GST_VIDEO_FPS_RANGE
-    ));
+    GST_STATIC_CAPS ("video/x-raw-yuv-strided, "
+        "format = (fourcc) NV12, "
+        "rowstride = (int) 4096, "
+        "width = " GST_VIDEO_SIZE_RANGE ", "
+        "height = " GST_VIDEO_SIZE_RANGE ", "
+        "framerate = " GST_VIDEO_FPS_RANGE));
 
 enum
 {
-- 
1.7.5.4

